### DISCUSSION TITLE
Password lock

### DISCUSSION THREAD

**Discussion by AndrewTXJ** [egzumer/uv-k5-firmware-custom#361] [Ideas]
Is it possible to add the password lock setting on the custom chirp? or maybe even implement it in to the menu, i modify the code to add back the password lock feature, but later i found out there is no way you can add a password except the original CPS, but since the original CPS is not compatible, i first set a password using the original CPS, the setting will all go wild, then i read the radio again from the custom egzumer chirp, then change back all the setting, this work, but i just wonder if it is possible to make it simple to just add it to the custom chirp or the menu.

**Comment by egzumer** [comment]
https://github.com/egzumer/uvk5-chirp-driver/discussions/13
